src/operations/search/mod.rs

```diff
@@ -776,9 +776,26 @@ pub struct SearchHitsHitsResult<T> {
     #[serde(rename="_routing")]
     pub routing: Option<String>,
     pub fields: Option<Value>,
-    pub highlight: Option<HighlightResult>
+    pub highlight: Option<HighlightResult>,
+    pub inner_hits: Option<HashMap<String, Value>>
```

Comment:
Is the `Value` needed due to the potential of more-than-one different type coming back in the results?  In which case its probably unavoidable.

Reply:
Indeed - You can have a look at this as reference: https://www.elastic.co/guide/en/elasticsearch/reference/current/search-request-inner-hits.html